tools/cloud-build/images/cluster-toolkit-dockerfile/README.md

```diff
@@ -0,0 +1,84 @@
+# Cluster Toolkit Dockerfile
+
+This repository contains a Dockerfile for building a Docker image with Cluster Toolkit and its dependencies installed and the `gcluster` binary readily available for use.
+
+## System Requirements
+
+* [Docker Engine](https://docs.docker.com/engine/) needs to be installed.
+
+## Environment Variables
+The following environment variables can be used to customize the build process:
```

Comment:
I see that the `docker build` command below sets these using `--build-arg` flags, so just want to confirm: if the user's machine has the environment variable `GO_VERSION=1.2.3` set, and they do not provide the `--build-arg GO_VERSION=???` flag, will the `GO_VERSION` value be set during the build to `1.2.3`?

If not, it'll probably be better to call these "build arguments" rather than "environment variables".

Reply:
If a user has GO_VERSION=1.2.3 set on their host machine and runs docker build without --build-arg GO_VERSION=..., the Dockerfile will use the default value for GO_VERSION. The host's GO_VERSION will not be used. To use the host's GO_VERSION, the user would need to explicitly pass it in using --build-arg GO_VERSION=$GO_VERSION. I'll update my wording to use "build arguments" so that it's more clear.